libretroshare/src/util/rsthreads.cc

```diff
@@ -194,6 +194,15 @@ bool RsThread::start(const std::string& threadName)
 			print_stacktrace();
 			return false;
 		}
+		if(!mTid)
```

Comment:
Is this compatible with windows? mTid is of class pthread_t and while it's a uint64_t on linux, it is a struct on windows.

Reply:
Appveyor build on windows successfully

Reply:
In win10 using MSys2:
@\libretroshare\src\util\rsthreads.h:276
	pthread_t mTid;
@C:\msys64\mingw32\i686-w64-mingw32\include\pthread.h:196
typedef uintptr_t pthread_t;
@C:\msys64\mingw32\i686-w64-mingw32\include\corecrt.h:82
typedef unsigned int uintptr_t;

So it is uint too...

Reply:
ok. My fault. I mixed it up with something else.